src/main/java/teammates/ui/pagedata/PageData.java

```diff
@@ -805,35 +810,6 @@ public String getPictureUrl(String pictureKey) {
                + Const.ParamsNames.USER_ID + "=" + account.googleId;
     }
 
-    public String getRecipientNames(Set<String> recipients, String courseId, String studentEmail, CourseRoster roster) {
```

Comment:
The removal of unused method is not related to this PR.

Reply:
Done